core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.directive.html

```diff
@@ -0,0 +1,188 @@
+            <em>The maximum length of subtopic title is 80 characters.</em>
+          </span>
+        </div>
+        <div class="subtopic-thumbnail">
+          <p>
+            <strong style="font-size: 1.2em;">Subtopic Thumbnail</strong>
+          </p>
+          <thumbnail-uploader filename="$ctrl.editableThumbnailFilename"
+                              update-filename="$ctrl.updateSubtopicThumbnailFilename"
+                              bg-color="$ctrl.editableThumbnailBgColor"
+                              update-bg-color="$ctrl.updateSubtopicThumbnailBgColor"
+                              allowed-bg-colors="$ctrl.allowedBgColors"
+                              aspect-ratio="'4:3'"
+                              preview-title="$ctrl.editableTitle"
+                              preview-description=""
+                              preview-description-bg-color="'#BE563C'"
+                              preview-footer="">
+          </thumbnail-uploader>
+        </div>
+        <div class="subtopic-html">
+          <div class="text-right" ng-if="$ctrl.schemaEditorIsShown">
+            <schema-based-editor schema="$ctrl.SUBTOPIC_PAGE_SCHEMA"
+                                 class="protractor-test-edit-subtopic-page-contents"
+                                 local-value="$ctrl.htmlData">
+            </schema-based-editor>
+            <button class="btn btn-success btn-sm protractor-test-save-subtopic-button" ng-click="$ctrl.updateHtmlData()"> Save
+            </button>
+          </div>
+          <div ng-if="!$ctrl.schemaEditorIsShown" ng-click="$ctrl.showSchemaEditor()" class="protractor-test-edit-html-content">
+            <span class="oppia-placeholder">
+              Give a description or explanation of the subtopic.
+              <i class="fa fa-pen"></i>
+            </span>
+            <angular-html-bind html-data="$ctrl.htmlData" class="protractor-test-subtopic-html-content">
+            </angular-html-bind>
+          </div>
+        </div>
+      </md-card>
+    </div>
+
+    <div class="subtopic-right-box">
+      <md-card class="oppia-subtopics-list-card topic-skill-count">
+        <div class="skill-count">
+          <span class="skill-count-value"><[$ctrl.subtopic.getSkillSummaries().length]></span>
+          <span class="skill-count-text">Skills</span>
+        </div>
+        <div class="create-topic-skill">
+          <button class="btn add-skill-btn" ng-click="$ctrl.createSkill()">
+            + ADD SKILL
+          </button>
+        </div>
+      </md-card>
+
+      <md-card class="skills-card protractor-test-subtopic-column"
+               dnd-list="$ctrl.subtopic.getSkillSummaries()"
+               dnd-drop="$ctrl.endMoveSkill($ctrl.subtopic.getId())">
+        <div class="subtopic-container">
+          <h3>Drop Uncategorized skill here to assign it to this subtopic.</h3>
+          <div class="skill-id protractor-test-subtopic-skillcard"
```

Comment:
Can you add a grey background color to the skill cards? Not sure what to click to drag now.
![image](https://user-images.githubusercontent.com/22347970/85408697-7a45eb80-b582-11ea-96d0-896c316e1d12.png)